client/src/components/App.jsx

```diff
@@ -180,47 +180,66 @@ class App extends React.Component {
     }
     let portfolio = [];
     let friends = [];
-    axios.get('/api/getPortfolio?username='+user)
-      .then((results) => {
-        portfolio = results.data;
-        axios.get('/api/getFriends?username='+user)
+    var rank = 0;
+    axios.get('/api/getRank', {
+      params: {
+        'username': user
+      }
+    })
+      .then((res) => {
```

Comment:
We will probably watch catch statements for all for all of the promises?  Or will the last catch take care of them all?

Reply:
I was under the impression that they would all be caught at the end, but could be wrong! I actually only added the getRank to this, the other calls were already there